benchmarks/src/main/scala/fs2/data/benchmarks/PrinterBenchmarks.scala

```diff
@@ -0,0 +1,85 @@
+package fs2.data.benchmarks
+
+import cats.effect.SyncIO
+import cats.effect.IO
+import fs2.data.json.Token
+import fs2.data.json.circe.*
+import fs2.{Fallible, Stream}
+import io.circe.Json
+import org.openjdk.jmh.annotations.*
+import org.openjdk.jmh.infra.Blackhole
+import cats.effect.unsafe.implicits.global
+
+import java.util.concurrent.TimeUnit
+
+@OutputTimeUnit(TimeUnit.MICROSECONDS)
+@BenchmarkMode(Array(Mode.AverageTime))
+@State(org.openjdk.jmh.annotations.Scope.Benchmark)
+@Fork(value = 1)
+@Warmup(iterations = 15, time = 5)
+@Measurement(iterations = 10, time = 5)
+class PrinterBenchmarks {
+
+  val intArrayStream =
+    Stream.emits(
+      Token.StartArray ::
+        (List
+          .range(0, 1000000)
+          .map(i => Token.NumberValue(i.toString())) :+ Token.EndArray))
```

Comment:
```suggestion
        (List.tabulate(1000000)(i => Token.NumberValue(i.toString())) :+ Token.EndArray))
```

Minor, but IMHO, that makes the intent a bit clearer.

I also would love to avoid appending to that long list, but I assume you want a single chunk? Otherwise using Stream's `++`, this could be simplified further.

Reply:
Yeah I want a single chunk, to see the overhead in processing one chunk.